roles/kubernetes-apps/helm/tasks/main.yml

```diff
@@ -17,6 +17,13 @@
   package:
     name: "{{ pyyaml_package }}"
     state: present
+  when: pyyaml_package is defined
+
+- name: Helm | Install PyYaml [flatcar]
+  include_tasks: pyyaml-flatcar.yml
+  when:
+    - pyyaml_package is undefined
+    - "'ID=flatcar' in os_release.stdout_lines"
```

Comment:
`  when: ansible_os_family in ["Flatcar", "Flatcar Container Linux by Kinvolk"]`
Could this be written like this ^ ? for consistency in the codebase